package.json

```diff
@@ -299,6 +302,7 @@
         "visible": true,
         "subPermissions": [
           "ui-orders.settings.order-templates.view",
+          "inventory-storage.holdings.collection.get",
```

Comment:
mb in third party? as we'll read holdings on details view

btw have you updated view?

Reply:
or we have separate ticket, I don't remember

Reply:
good question. nothing about details view in the descriptions

Reply:
ping Dennis in ticket to create ticket for details, but permissions should be applied for third party perms set